src/index.ts

```diff
@@ -12,6 +12,8 @@ const log = new Log('App index')
 
 window.checker = new Checker()
 
+const gateways = Object.keys(report)
+
```

Comment:
Note: in the frontend we "just" change the source of truth for the list of gateways and drop the `/ipfs/:hash`.